scio-extra/src/main/scala/com/spotify/scio/extra/annoy/AnnoyUri.scala

```diff
@@ -60,10 +60,12 @@ private class LocalAnnoyUri(val path: String) extends AnnoyUri {
 }
 
 private class RemoteAnnoyUri(val path: String, options: PipelineOptions) extends AnnoyUri {
-  private[this] val rfu: RemoteFileUtil = RemoteFileUtil.create(options)
+  {
+    RemoteFileUtil.configure(options)
+  }
```

Comment:
I'm not sure we should do this here but let the user initialize outside the model